src/main/resources/liquibase.properties

```diff
@@ -1,4 +1,4 @@
-url=jdbc:mysql://localhost:3306/rest?serverTimeZone=UTC
+url=jdbc:mysql://localhost:3306/bookstore?serverTimeZone=UTC
 username=root
 password=---SQLdno12
```

Comment:
I would change it to some neutral text because this is a rather debatable topic :)

Reply:
I couldn't find a way to hide this into env var.((